debian/autobake-deb.sh

```diff
@@ -134,12 +145,13 @@ in
     ;&
   "jammy"|"kinetic")
     add_lsb_base_depends
-    ;&
+    ;;
```

Comment:
this is changing the behaviour of the fall throughs.

Recommend:
1. leave this as `;&`. Remove `kinetic` as its EOL. Add `change_compat 11`.

https://mariadb.com/kb/en/mariadb-platform-deprecation-policy/